[2024-07-22T00:12:56.299+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-22T00:12:56.346+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-07-22T00:12:54.774733+00:00 [queued]>
[2024-07-22T00:12:56.361+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-07-22T00:12:54.774733+00:00 [queued]>
[2024-07-22T00:12:56.362+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-22T00:12:56.384+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-07-22 00:12:54.774733+00:00
[2024-07-22T00:12:56.392+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=794) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-22T00:12:56.394+0000] {standard_task_runner.py:63} INFO - Started process 795 to run task
[2024-07-22T00:12:56.394+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'manual__2024-07-22T00:12:54.774733+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmppwoi283m']
[2024-07-22T00:12:56.396+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask first_task
[2024-07-22T00:12:56.458+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v2.first_task manual__2024-07-22T00:12:54.774733+00:00 [running]> on host eefce25a52e3
[2024-07-22T00:12:56.565+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-22T00:12:54.774733+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-22T00:12:54.774733+00:00'
[2024-07-22T00:12:56.566+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-22T00:12:56.581+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-22T00:12:56.582+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello World, this is the first task to be done']
[2024-07-22T00:12:56.594+0000] {subprocess.py:86} INFO - Output:
[2024-07-22T00:12:56.595+0000] {subprocess.py:93} INFO - Hello World, this is the first task to be done
[2024-07-22T00:12:56.596+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-22T00:12:56.597+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-22T00:12:56.624+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=manual__2024-07-22T00:12:54.774733+00:00, execution_date=20240722T001254, start_date=20240722T001256, end_date=20240722T001256
[2024-07-22T00:12:56.649+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-22T00:12:56.674+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-22T00:12:56.679+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
